### PR TITLE
Coin system

### PR DESCRIPTION
**Pull request by TanJinWen-NP**
My apologies for the misunderstanding, please refer to the updated code. :)

### CODE REVIEW

File: SDD-assg2/Program.cs [hunk 1]
```diff
@@ -7,7 +7,9 @@
 List<List<Building?>> game_field = new List<List<Building?>>();
 List<Building> list_of_Buildings = new List<Building>(); // contains the list of buildings that can be selected
 // turn counter
-int turn = 0;
```
Comment (warrenkenzie) [inline SDD-assg2/Program.cs:10]: conflict with turn

File: SDD-assg2/Program.cs [hunk 2]
```diff
@@ -116,7 +127,7 @@
     } while (If_have_exceptions); // Continue the loop until no exception occur (which means chooses correct building)
 
     // THE REST OF THE GAME LOOP    
-    while (true)
```
Comment (warrenkenzie) [inline SDD-assg2/Program.cs:119]: conflict with main branch

**Review by warrenkenzie — COMMENTED** [review]
some conflicts, please resolve them

File: SDD-assg2/Program.cs (hunk 1 above)
Comment (TanJinWen-NP) [reply SDD-assg2/Program.cs:8]: Resolved

File: SDD-assg2/Program.cs (hunk 2 above)
Comment (TanJinWen-NP) [reply SDD-assg2/Program.cs:118]: Resolved by removing the previous variable

**Review by warrenkenzie — APPROVED** [review]
conflicts resolved correctly

**Pull request merged**